guides/templates/guides/guide_list.html

```diff
@@ -18,6 +18,14 @@ <h2 class="float-left">Guides</h2>
   </div>
   <p class="large">Here we collect community-driven guides and tutorials for programming, our server, and more. Want to add something? Feel free to write your own!</p>
   <h3>New Guides</h3>
+  <hr>
+  <form action="{% url 'guides:search' %}" method="GET">
+      <div id="search-key">
+        <input type="text" placeholder="Search guides" name="term">
+      </div>
+      <input type="submit" value="Search" id="search-button">
+  </form>
+  <hr>
```

Comment:
I feel like the `<h3>New Guides` should be below this. Right now we have:
Header
< search bar >
Content

which is a bit weird, because from the header, you'd expect content directly afterwards. I think it would be nicer if we could move this above the header, and possibly remove one of the `<hr>`s. This is what I'd suggest:
![image](https://user-images.githubusercontent.com/22679924/41378829-9b58062e-6f60-11e8-955b-874b91d9c34a.png)

With some of the margin removed, this will look ideal, imo.